testnet/stacks-node/src/neon_node.rs

```diff
@@ -375,92 +384,82 @@ fn try_mine_microblock(
     chainstate: &mut StacksChainState,
     sortdb: &SortitionDB,
     mem_pool: &MemPoolDB,
-    winning_tip_opt: Option<&(ConsensusHash, BlockHeaderHash, Secp256k1PrivateKey)>,
+    winning_tip: (ConsensusHash, BlockHeaderHash, Secp256k1PrivateKey),
 ) -> Result<Option<StacksMicroblock>, NetError> {
+    let ch = winning_tip.0;
```

Comment:
style nit: pattern match whole tuple at once

Reply:
Rust doesn't let you destruct tuples if the lhs is a `let`-binding.